website/content/docs/rfcs/opentelemetry.mdx

```diff
@@ -0,0 +1,96 @@
+# Switch to OpenTelemetry
+
+## Summary
+
+Should we drop [`go-metrics`][] library and thereby support for the metrics
```

Comment:
This is a level of sorcery I've not yet seen :laughing: Didn't realize you could leave the link blank and have it auto-fill to the text value.